src/test/duke/task/DeadlineTest.java

```diff
@@ -0,0 +1,21 @@
+package duke.task;
+
+import org.junit.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.Assert.assertEquals;
+
+public class DeadlineTest {
+    @Test
+    public void deadlineTest() {
+        assertEquals(new Deadline("return book", LocalDate.parse("2019-10-15")).toString(), "[D][ ] return book (by: Oct 15 2019)");
```

Comment:
This line has exceeded the limit of 120 words. Would it be better to separate it into 2 lines instead?